extract.sh

```diff
@@ -128,17 +128,20 @@ function extract {
         fi
         (cd "${in_dir}" && ar "${VERBOSE_FLAG[*]}"x "$n")
       ;;
-      *.7z|*.arj|*.cab|*.cb7|*.chm|*.deb|*.dmg|*.iso|*.lzh|*.msi|*.pkg|*.rpm|*.udf|*.wim|*.xar)
+      *.7z|*.arj|*.cab|*.cb7|*.chm|*.deb|*.dmg|*.iso|*.lzh|*.msi|*.pkg|*.rpm|*.udf|*.wim|*.xar|*.exe|*.apk)
```

Comment:
*.exe, *.apk is not an archive, other tools are used to work on them

Reply:
I am ... not sure what do you mean by that?

They are still extractable.